src/cryptography/encrypt.js

```diff
@@ -133,10 +133,33 @@ const getTagBuffer = tag => {
 	return tagBuffer;
 };
 
-const decryptAES256GCMWithPassword = (
-	{ iterations = PBKDF2_ITERATIONS, cipherText, iv, salt, tag },
-	password,
-) => {
+const validateEncryptedPassphrase = ({ cipherText, iv, salt, tag }) => {
+	const requiredProperties = [
+		[cipherText, 'Cipher text'],
+		[iv, 'IV'],
+		[salt, 'Salt'],
+		[tag, 'Tag'],
+	];
```

Comment:
I would prefer to use Map instead of nested array. It'd save some memory since it doesn't have a prototype. see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map `Iterating Maps with for..of` section